packages/dotnet/Readme/HarJsonTranslationLogics/ReadmeApiCaller.cs

```diff
@@ -6,38 +6,32 @@
 
 namespace Readme.HarJsonTranslationLogics
 {
-    class ReadmeApiCaller
```

Comment:
Is there a linting system for .NET that we can incorporate?

Reply:
Not sure... ticketed here: https://github.com/readmeio/metrics-sdks/issues/453

When you install the official C# plugin for VSCode it does quite a few tidy ups (mostly whitespace) but would be nice to find something automated.